packages/gatsby/src/utils/dev-ssr/render-dev-html.ts

```diff
@@ -131,11 +131,11 @@ export const renderDevHTML = ({
       isClientOnlyPage = true
     }
 
-    const { boundActionCreators } = require(`../../redux/actions`)
-    const { createServerVisitedPage } = boundActionCreators
+    const { actions } = require(`../../redux/actions`)
+    const { createServerVisitedPage } = actions
     // Record this page was requested. This will kick off adding its page
     // component to the ssr bundle (if that's not already happened)
-    createServerVisitedPage(pageObj.componentChunkName)
+    internalStore.dispatch(createServerVisitedPage(pageObj.componentChunkName))
```

Comment:
Why aren't we using the store provided by `renderDevHTML`?

Reply:
Didn't have a specific reason, just mimicked what I did in other places. Can change it to use the provided store 👍